app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,64 @@
+class RecipeFoodsController < ApplicationController
+  before_action :set_recipe_food, only: %i[show edit update destroy]
+
+  # GET /recipe_foods or /recipe_foods.json
+  def index
+    @recipe_foods = RecipeFood.all.includes(:recipe, :food)
+  end
+
+  def show
+    @recipe_food = RecipeFood.includes(:recipe, :food).find(params[:id])
+  end
+
+  # GET /recipe_foods/new
+  def new
+    @recipe_food = RecipeFood.new
+  end
+
+  def edit; end
+
+  def create
+    @recipe_food = RecipeFood.new(recipe_food_params)
+
+    respond_to do |format|
+      if @recipe_food.save
+        format.html { redirect_to recipe_food_url(@recipe_food), notice: 'Recipe food was successfully created.' }
+        format.json { render :show, status: :created, location: @recipe_food }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @recipe_food.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def update
+    respond_to do |format|
+      if @recipe_food.update(recipe_food_params)
+        format.html { redirect_to recipe_food_url(@recipe_food), notice: 'Recipe food was successfully updated.' }
+        format.json { render :show, status: :ok, location: @recipe_food }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @recipe_food.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def destroy
+    @recipe_food.destroy
+
+    respond_to do |format|
+      format.html { redirect_to recipe_foods_url, notice: 'Recipe food was successfully destroyed.' }
+      format.json { head :no_content }
+    end
+  end
+
+  private
+
+  def set_recipe_food
+    @recipe_food = RecipeFood.includes(:recipe, :food).find(params[:id])
+  end
+
+  def recipe_food_params
+    params.require(:recipe_food).permit(:quantity, :recipe_id, :food_id)
+  end
+end
```

Comment:
When creating a recipe, I am not able to add the food from the foodlist. If i have some food items in my food list, they automatically show as food items under the recipee. I have created [this ](https://www.loom.com/share/c15d089a799e485bab9efb56c61429dd?sid=15602e16-d36e-46d1-a72b-5eac8750963a)video where i explained what the issue is

Reply:
> When creating a recipe, I am not able to add the food from the foodlist. Instead it creats a new food item which then is visible in all the recipes. And If I have some food items in my food list, they automatically show as food items under the recipe. I have created [this ](https://www.loom.com/share/c15d089a799e485bab9efb56c61429dd?sid=15602e16-d36e-46d1-a72b-5eac8750963a)video where i explained what the issue is

[Based on this requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=If%20the%20user%20is%20the%20owner%20of%20the%20recipe%2C%20should%20lead%20to%20the%20form%20that%20allows%20the%20user%20to%20add%20new%20food), I think the button should lead to the form that allows the user to add new food. This new food is added to the users food list.